app/views/users/require_2sv.html.erb

```diff
@@ -1,16 +1,23 @@
-<% content_for :title, "Create new user" %>
+<% content_for :title, "2-step verification settings for new user" %>
 
-<h1>2-step verification settings for new user</h1>
-
-<%= form_for @user, :html => {:class => 'well'} do |f| %>
+<%= form_for @user do |f| %>
   <%= f.hidden_field :skip_update_user_permissions, value: "true" %>
-  <p class="checkbox">
-    <%= f.label :require_2sv do %>
-      <%= f.check_box :require_2sv %> Mandate 2-step verification for this user <%= "(this will remove their exemption)" if @user.exempt_from_2sv? %>
-    <% end %>
-    <br/>
-    User will be prompted to set up 2-step verification again the next time they sign in.
-  </p>
+  <%= render "govuk_publishing_components/components/hint", {
```

Comment:
Minor: Did you consider using the `hint_text` option on the `checkboxes` component? It's not very clear, but [this code](https://github.com/alphagov/govuk_publishing_components/blob/cb400fd0526e4f796a978ed3a932f1e1c9e05a15/lib/govuk_publishing_components/presenters/checkboxes_helper.rb#L40C10-L40C19) suggests it's an available option...